src/js/locale/pl.yaml

```diff
@@ -0,0 +1,532 @@
+      description: Przesuń i upuść aby zmienić priorytety dostawców
+    reporting:
+      label: Raportowanie
+      sublabel: Włącz anonimowe raportowanie statystyk
+      description: 'Te anonimowe informacje są dla nas ważne w celu identyfikacji błędów i potencjalnych nowych funkcji. Pomagają nam rozwijać Iris i sprawiać by stawało się coraz lepsze. Chcesz wiedzieć więcej? Przeczytaj '
+      privacy_policy: politykę prywatności
+      are_you_sure: Na pewno nie chcesz nas wesprzeć?
+    commands:
+      label: Komendy
+  advanced:
+    title: Zaawansowane
+    start_local_scan: Rozpocznij lokalne skanowanie
+    artist_uri:
+      label: URI wykonawców w bibliotece
+      description: URI używane do organizacji wykonawców w bibliotece
+    album_uri:
+      label: URI albumów w bibliotece
+      description: URI używane do organizacji albumów w bibliotece
+    connections:
+      label: Połączenia
+    version:
+      label: Wersja
+      upgrade_available: 'Dostępna aktualizacjia (%{version})'
+      upgrade: 'Zaktualizuj do %{version}'
+      up_to_date: Najnowsza wersja
+    share_configuration: Udostępnij konfigurację
+    restart: Restartuj serwer
+    reset_cache: Resetuj pamięć podręczną
+    reset: Resetuj wszystkie ustawienia
+  about:
+    title: O projekcie
+    blurb_1: ' jest otwartoźródłowym projektem prowadzonym przez '
+    blurb_2: '. Jest dostarczane bez jakichkolwiek gwarancji. Jeśli zapłaciłeś komukolwiek za to oprogramowanie, skontaktuj się z nami.'
+    donate: Wesprzyj
+    github: GitHub
+  pusher_connection_list:
+    not_connected: Rozłączono
+    no_connections: Brak połączeń
+    you: ' (ty)'
+  servers:
+    inactive: Nieaktywny
+    connecting: Łączenie
+    disconnected: Rozłączono
+    connected: Połączono
+    new_server: Nowy serwer
+    name: Nazwa
+    host: Host
+    port: Port
+    encryption:
+      label: Szyfrowanie
+      sublabel: Używaj SSL
+      description: Wymaga proxy SSL
+    switch: Przełącz do tego serwera
+    reconnect: Połącz ponownie
+  services:
+    title: Usługi
+    limited_access: Ograniczony dostęp
+    unknown: Nieznany
+    current_user: Aktualny użytkownik
+    authorization: Autoryzacja
+    refresh_token: Odśwież token
+    authorized: Zautoryzowano
+    read_only: Tylko do odczytu
+    unauthorized: Niezautoryzowano
+    enabled: Odblokowany
+    disabled: Zablokowany
+    connected: Połączony
+    disconnected: Rozłączony
+    spotify:
+      mopidy_spotify_not_running: Rozszerzenie Mopidy-Spotify nie jest uruchomione - nie będziesz mógł odtwarzać ścieżek ze Spotify
+      country:
+        label: Kraj
+        description: Kod kraju ISO 3166-1 alpha-2 (np. 'PL')
+      locale:
+        title: Lokalizacja
+        description: Kod językowy ISO 639 oraz ISO 3166-1 alpha-2 połączone podkreślnikiem (np. 'pl_PL')
+    icecast:
+      enable:
+        label: Odblokowane
+        description: Strumieniuj audio do tej przeglądarki
+      location:
+        label: Lokacja

```

Comment:
Debatable
https://sjp.pwn.pl/poradnia/haslo/lokacja-i-lokalizacja;12628.html

Reply:
As I understand "lokalizacja" is more related to some place in terrain, while "lokacja" describes placement in possibly some abstract space. I would go for the latter, but I am not pretty much sure.

Reply:
I'm not sure either. Up to you.

Reply:
I'd leave it as it is. The difference is really blurry anyway